### PR TITLE
refactor: rename hub-auth to auth

### PR DESCRIPTION
**Pull request by rolsonquadras**
- go import
- docker pkg names
- env var

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: ci/version_var.sh [hunk 1]
```diff
@@ -25,5 +25,5 @@ else
   PROJECT_PKG_REPO=${RELEASE_REPO}
 fi
 
-export HUB_AUTH_TAG=$PROJECT_VERSION
-export HUB_AUTH_PKG=${PROJECT_PKG_REPO}/${BASE_AUTH_PKG_NAME}
+export AUTH__TAG=$PROJECT_VERSION
```
Comment (birtony) [inline ci/version_var.sh:28]: Is this expected to have double underscores in the name of this variable?

File: test/bdd/pkg/bootstrap/steps.go [hunk 2]
```diff
@@ -14,13 +14,13 @@ import (
...
-	bddctx "github.com/trustbloc/hub-auth/test/bdd/pkg/context"
-	"github.com/trustbloc/hub-auth/test/bdd/pkg/login"
+	"github.com/trustbloc/auth/pkg/restapi/operation"
+	bddctx "github.com/trustbloc/auth/test/bdd/pkg/context"
+	"github.com/trustbloc/auth/test/bdd/pkg/login"
 )
 
 const (
-	bootstrapDataPath = login.HUB_AUTH_HOST + "/bootstrap"
+	bootstrapDataPath = login.AUTH__HOST + "/bootstrap"
```
Comment (birtony) [inline test/bdd/pkg/bootstrap/steps.go:23]: And here?

File: test/bdd/pkg/login/steps.go [hunk 3]
```diff
@@ -18,15 +18,15 @@ import (
 	"github.com/cucumber/godog"
 	"github.com/google/uuid"
 
-	bddctx "github.com/trustbloc/hub-auth/test/bdd/pkg/context"
+	bddctx "github.com/trustbloc/auth/test/bdd/pkg/context"
 )
 
 const (
-	HUB_AUTH_HOST                   = "https://localhost:8070"
+	AUTH__HOST                      = "https://localhost:8070"
```
Comment (birtony) [inline test/bdd/pkg/login/steps.go:25]: Same here

File: test/bdd/pkg/login/steps.go [hunk 4]
```diff
@@ -18,15 +18,15 @@ import (
...
 	hubAuthHydraAdminURL            = "https://localhost:4445"
 	hubAuthOIDCProviderURL          = "https://localhost:4444/"
-	hubAuthOIDCProviderSelectionURL = HUB_AUTH_HOST + "/ui"
-	hubAuthSelectOIDCProviderURL    = HUB_AUTH_HOST + "/oauth2/login"
+	hubAuthOIDCProviderSelectionURL = AUTH__HOST + "/ui"
+	hubAuthSelectOIDCProviderURL    = AUTH__HOST + "/oauth2/login"
```
Comment (birtony) [inline test/bdd/pkg/login/steps.go:29]: ^

File: test/bdd/pkg/secrets/mock_key_server.go [hunk 5]
```diff
@@ -11,22 +11,23 @@ import (
...
 	"io/ioutil"
 	"net/http"
+
+	"github.com/trustbloc/auth/pkg/restapi/operation"
 )
 
 func NewMockKeyServer(token string, tlsConfig *tls.Config) *MockKeyServer {
 	return &MockKeyServer{
-		ApiToken: token,
+		ApiToken:  token,
```
Comment (birtony) [inline test/bdd/pkg/secrets/mock_key_server.go:22]: 🤔

File: test/bdd/pkg/secrets/steps.go [hunk 6]
```diff
@@ -11,13 +11,13 @@ import (
...
-	"github.com/trustbloc/hub-auth/test/bdd/pkg/login"
+	"github.com/trustbloc/auth/test/bdd/pkg/login"
 
-	bddctx "github.com/trustbloc/hub-auth/test/bdd/pkg/context"
+	bddctx "github.com/trustbloc/auth/test/bdd/pkg/context"
 )
 
 const (
-	secretsEndpoint = login.HUB_AUTH_HOST + "/secret"
+	secretsEndpoint = login.AUTH__HOST + "/secret"
```
Comment (birtony) [inline test/bdd/pkg/secrets/steps.go:20]: ^

File: ci/version_var.sh (hunk 1 above)
Comment (rolsonquadras) [reply ci/version_var.sh]: [fixed](https://github.com/trustbloc/auth/compare/fb097da03b60b7d5ef85577771b66193a48dca0d..044a75a29b1020729c29a939c0216288dce4ffae)

File: test/bdd/pkg/bootstrap/steps.go (hunk 2 above)
Comment (rolsonquadras) [reply test/bdd/pkg/bootstrap/steps.go]: [fixed](https://github.com/trustbloc/auth/compare/fb097da03b60b7d5ef85577771b66193a48dca0d..044a75a29b1020729c29a939c0216288dce4ffae)

File: test/bdd/pkg/login/steps.go (hunk 3 above)
Comment (rolsonquadras) [reply test/bdd/pkg/login/steps.go]: [fixed](https://github.com/trustbloc/auth/compare/fb097da03b60b7d5ef85577771b66193a48dca0d..044a75a29b1020729c29a939c0216288dce4ffae)

File: test/bdd/pkg/login/steps.go (hunk 4 above)
Comment (rolsonquadras) [reply test/bdd/pkg/login/steps.go]: [fixed](https://github.com/trustbloc/auth/compare/fb097da03b60b7d5ef85577771b66193a48dca0d..044a75a29b1020729c29a939c0216288dce4ffae)

File: test/bdd/pkg/secrets/steps.go (hunk 6 above)
Comment (rolsonquadras) [reply test/bdd/pkg/secrets/steps.go]: [fixed](https://github.com/trustbloc/auth/compare/fb097da03b60b7d5ef85577771b66193a48dca0d..044a75a29b1020729c29a939c0216288dce4ffae)

**Review by birtony — APPROVED** [review]

**Review by Moopli — APPROVED** [review]

**Review by sudeshrshetty — APPROVED** [review]

**Pull request merged**